e2e/Utils.ts

```diff
@@ -101,16 +101,12 @@ export const makeTemplate =
     });
 
 export const cleanup = (directory: string) => {
```

Comment:
Can put this in test-utils perhaps?

https://github.com/jestjs/jest/tree/main/packages/test-utils

Reply:
Yes, ai need to refactor, but use maxRetry don't works
 Do you have any ideia?